src/wh_11_1/Test.java

```diff
@@ -0,0 +1,109 @@
+package wh_11_1;
+
+import java.io.*;
+import java.nio.file.*;
+
+class Test {
+    public static void main(String[] args) {
+        final String FILE_NAME = "TestFile.txt";
+        final String RESULTS_FILE_NAME = "testResults.txt";
+        try {
+            Employe[] employes = EmoloyeeReader.readFromFile(FILE_NAME);
+            for (Employe x : employes) {
+                System.out.println(x);
+            }
+            fileChecker(RESULTS_FILE_NAME);
+
+            fileWriter(RESULTS_FILE_NAME, avgEmployeeSalary(employes));
+            fileWriter(RESULTS_FILE_NAME, minEmployeeSalary(employes));
+            fileWriter(RESULTS_FILE_NAME, maxEmployeeSalary(employes));
+            fileWriter(RESULTS_FILE_NAME, empNOinDepartment(employes, "it"));
+
+            fileReader(RESULTS_FILE_NAME);
+        } catch (FileNotFoundException ex) {
+            System.err.println(ex.getMessage());
+        }
+    }
+
+    private static String avgEmployeeSalary(Employe[] employes) {
+        int counter = 0;
+        double salarySum = 0;
+        for (Employe employe : employes) {
+            salarySum += employe.getPayment();
+            counter++;
+        }
+        double avgValue = salarySum / counter;
+        return "Średnia wypłata:" + avgValue;
+    }
+
+    private static String minEmployeeSalary(Employe[] employes) {
+        Employe minSalary = employes[0];
```

Comment:
program nie zadziala dla pustej tablicy

Reply:
zmienilem:
`        try {
            Employe minSalary = employees[0];
            for (Employe employe : employees) {
                if (employe.getPayment() < minSalary.getPayment())
                    minSalary = employe;
            }
            return "najmniejsza wypłata: " + minSalary.getPayment();
        } catch (ArrayIndexOutOfBoundsException ex) {
            return "Nie da sie policzyc. Pusta tablica";
        }`